docker/Dockerfile-ubuntu-20.04

```diff
@@ -1,44 +1,12 @@
 FROM ubuntu:focal
 ENV DEBIAN_FRONTEND noninteractive
 COPY build-and-install-scafacos.sh /tmp/
-RUN apt-get update && apt-get install -y \
-    apt-utils \
-    cmake \
-    ccache \
-    autoconf \
-    gfortran \
-    build-essential \
-    pkg-config \
-    openmpi-bin \
-    libopenmpi-dev \
-    clang-9 \
-    clang-tidy-9 \
-    clang-format-9 \
-    libfftw3-dev \
-    libboost-dev libboost-serialization-dev libboost-mpi-dev libboost-filesystem-dev libboost-test-dev \
-    libgsl-dev \
-    cython3 \
-    python3 \
-    python3-numpy \
-    python3-scipy \
-    python3-h5py \
-    python3-vtk7 \
-    python3-pip \
```

Comment:
but we need pip in this image

Reply:
```
/bin/sh: 1: pip3: not found
Pull request: not pushing to registry.
```
Yet the checkmark is green...

Reply:
where is it green?
https://github.com/KaiSzuttor/docker/runs/557442779?check_suite_focus=true

Reply:
ah that's weird

Reply:
https://github.com/espressomd/docker/runs/557443033